SmartPlaces.Facilities/lib/IngestionManager/src/RedisTwinMappingIndexer.cs

```diff
@@ -7,8 +7,8 @@
 namespace Microsoft.SmartPlaces.Facilities.IngestionManager
 {
     using System.Net.Sockets;
+    using System.Reflection.Metadata.Ecma335;
```

Comment:
what is this one for?

Reply:
Woops, that I've no idea about. Removed it and updated PR. Thanks for catching it